src/functions-and-arrays.js

```diff
@@ -52,16 +86,35 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(wordsUnique) {
+  if (wordsUnique.length === 0) {
+    return null;
+  }
 
+  wordsUnique.forEach((word) => {});
+}
```

Comment:
one possible solution would be like this:
```
function uniquifyArray(wordsUnique) {
  if (wordsUnique.length === 0) {
    return null;
  }

  let unique = []
  wordsUnique.forEach((word) => {
    if (unique.indexOf(word) < 0){
      unique.push(word)
    }
  });
  return unique
}
```

Reply:
If you have any questions about this task or the bonus tasks, please feel free to ask me (Joanne) or Marian and Karina